gcc4mbed/external/mbed/libraries/mbed/targets/hal/TARGET_NXP/TARGET_LPC43XX/PeripheralNames.h

```diff
@@ -45,7 +48,23 @@ typedef enum {
     ADC1_4,
     ADC1_5,
     ADC1_6,
-    ADC1_7
+    ADC1_7,
```

Comment:
Is it possible to isolate these changes to the Smoothie specific source files (in a derived class for example) or do they need to be modified in the core mbed sources? If they need to be done in the mbed SDK sources then a PR should be issued to get them into https://github.com/ARMmbed/mbed-os or they will be lost the next time we refresh the gcc4mbed portion of the source tree.